Codeinwp/feedzy-rss-feeds
# feat: move translation option to action tab

### Summary

This moves the translation settings from Import Metabox to the Translation Action of the Title/Content tag.

### Will affect visual aspect of the product

YES

### Screenshots 

<img width="1072" alt="Screenshot 2024-12-10 at 3 54 40 AM" src="https://github.com/user-attachments/assets/8093e342-2e29-4d4a-8e94-0b14d5ae68bd">


### Test instructions


- This PR requires thorough testing.
- First, make sure translations work properly with the new options.
- Second, we need to confirm backward compatibility testing in two ways:
- If you have set translation settings before updating to this, they should continue to work as normal.
- If you have set translation settings before updating to this, you should be easily able to edit them from the action bar and it should inherit the language that was previously set.

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/feedzy-rss-feeds-pro/issues/784.
